src/Components/SingleArticle.jsx

```diff
@@ -108,13 +149,23 @@ export const SingleArticle = () => {
 
       {comments.map((comment) => {
         const date = comment.created_at
-          ? new Date(comment.created_at).toUTCString()
-          : "now";
+          ? timeAgo.format(new Date(comment.created_at))
+          : comment.posted;
         return (
           <div key={comment.comment_id ? comment.comment_id : comment.tempKey}>
             <p key={comment.author} className="commentAuthor">
               {comment.author}
             </p>
+            {comment.comment_id && comment.author === "jessjelly" && (
```

Comment:
Hmmm... this isn't technically wrong, but I wouldn't hate it if you had, for example, jessJelly saved somewhere as a 'user' variable and passed in here. Even if it's just within the same page, it would be a nice nod to the fact that this wouldn't typically be hardcoded. Bonus points if you use user context.

Reply:
cool, will add that now